### PR TITLE
created third section and fixed some moments in the header and the fi…

### PR DESCRIPTION
**Pull request by code-by-iryna**
…rst section

### CODE REVIEW

File: src/components/AppHeader/index.scss [hunk 1]
```diff
@@ -15,13 +15,21 @@
...
-        height: 100%;
-        border-radius: 25px;
     }
 
     &__actions {
         display: flex;
         gap: 40px;
     }
+
+    &__button-action {
```
Comment (stasyurkanis) [inline src/components/AppHeader/index.scss:25]: I see that you already styled the button here https://github.com/Irina-Suprun/new-coffee/pull/3/files#diff-6d1ac1e7072d002fd68fa47aec09fccdbdd03bfc6b3366e3a7bd5fd96239abf3R69

It seems that you only need a restyled size, don't you?

File: src/styles/index.scss [hunk 2]
```diff
@@ -64,17 +66,20 @@ img {
...
+.button {
     display: inline-flex;
     justify-content: center;
     align-items: center;
     cursor: pointer;
     border: none;
-    background-color: transparent;
+    width: 174px;
+    height: 46px;
+    border-radius: 50px;
```
Comment (stasyurkanis) [inline src/styles/index.scss:77]: I suppose here should be 50%, right?

File: src/components/Sections/OrderSection/index.vue [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    <section class="order-section section">
+        <div class="container">
+            <div class="order-section__title-box">
+                <strong class="title">Check our sales for popular products</strong>
+                <a href="" class="order-section__sales-link link">All sales</a>
+            </div>
+            <ul class="order-section__list list">
+                <li class="order-section__item">
+                    <div class="order-section__discount">
+                        <p>-30%</p>
```
Comment (stasyurkanis) [inline src/components/Sections/OrderSection/index.vue]: such places look more like span

File: src/components/Sections/OrderSection/index.vue [hunk 4]
```diff
@@ -0,0 +1,89 @@
+<template>
+    <section class="order-section section">
+        <div class="container">
+            <div class="order-section__title-box">
+                <strong class="title">Check our sales for popular products</strong>
+                <a href="" class="order-section__sales-link link">All sales</a>
```
Comment (stasyurkanis) [inline src/components/Sections/OrderSection/index.vue]: if there is more than 1 attribute, it is customary to write it down one by one

`<a 
    href=""
    class="order-section__sales-link link"
 >
    All sales
</a>`

File: src/components/AppHeader/index.scss (hunk 1 above)
Comment (code-by-iryna) [reply src/components/AppHeader/index.scss:25]: I thought that if I don't use all rule I need to create new

Comment (code-by-iryna) [reply src/components/AppHeader/index.scss:25]: I`m editing it

File: src/styles/index.scss (hunk 2 above)
Comment (code-by-iryna) [reply src/styles/index.scss:77]: Realy, it's my mistake

Comment (code-by-iryna) [reply src/styles/index.scss:77]: although, 50% make wrong picture and 50px make good

Comment (code-by-iryna) [reply src/styles/index.scss:77]: Why can't I use 50px?

File: src/components/Sections/OrderSection/index.vue (hunk 3 above)
Comment (code-by-iryna) [reply src/components/Sections/OrderSection/index.vue]: Ok, I'll edit it

File: src/components/Sections/OrderSection/index.vue (hunk 4 above)
Comment (code-by-iryna) [reply src/components/Sections/OrderSection/index.vue]: ok, thanks

**Pull request merged**